crates/testfile/src/lib.rs

```diff
@@ -467,7 +467,7 @@ mySender = \"0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266\"
 [spam.tx]
 from = \"{mySender}\"
 to = \"{_sender}\"
-value = \"1eth\"
+value = \"1000000000000000000\"
```

Comment:
this change will have to be reverted when we add the rest of the functionality from https://github.com/flashbots/contender/issues/387 -- remember we want to be able to specify "1 eth" as the value instead of "1000000000000000000"